### DISCUSSION TITLE
Layer Transform Tool

### DISCUSSION THREAD

**Discussion by wwderw** [Orama-Interactive/Pixelorama#662] [Ideas]
I was wondering if it would be possible to have the ability to transform a layer by itself.  Also with a user defined pivot point as well.  Something like the move tool, but with the extra functionality of being able to rotate (and change the point of rotation, but with the ability to quickly go back to the layer's "center"), scale, shear etc.

I'm thinking of being able to quickly get animation roughed out in the more or less sketching phase or if the users is trying to do more or less cut out animation style.

**Comment by negaspace** [comment]
Was about to add this request in my own words, though I had not thought about rotate/shearing functions: 

The idea is to have a kind of layer (possibly just an additional function for group layers) that can just control for position in space, so that the underlying cel drawing doesn't need to be adjusted for every frame of movement. So if I had an object, say a floating glowing ball, that ideally would not change on a drawing level, I could just move it around on one layer. Then if I needed to change it to a glowing triangle through all frames, I would just edit the single cel drawing without having to redo the movement. This would also be helpful for testing walk/run cycles, allowing the drawing sequence to repeat underneath while moving the character in space on another layer. 

This is an idea from the world of vector animation like symbols in Flash or pegs (a better model) in Toonboom.
